portfolio/src/main/webapp/css/about.css

```diff
@@ -0,0 +1,59 @@
+.blackBackgroundImage, .longGroveImage, .stanfordImage, .hobbiesImage {
+  position: relative;
+  opacity: 1;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+.blackBackgroundImage {
```

Comment:
We should be lower-casing class names:
https://google.github.io/styleguide/htmlcssguide.html#Capitalization